p2p/src/peer_manager/mod.rs

```diff
@@ -1637,6 +1637,8 @@ where
             // might establish only a few OutboundBlockRelay connections and then stop establishing
             // new connections until both it's out of IBD and its tip becomes stale (following
             // the second part of the condition above).
+            // TODO: dns seeds are considered bad sources of peer addresses, because they can be
+            // compromised. It's better to avoid connecting to them unless necessary.
             true
```

Comment:
Please create an issue about this, explaining the problem in more detail.

Reply:
I've described it in an additional item in the TODO section of https://github.com/mintlayer/mintlayer-core/issues/832